test/e2e-olm/utils.go

```diff
@@ -124,30 +97,17 @@ func createElasticsearchCR(t *testing.T, f *test.Framework, ctx *test.Context, e
 		},
 	}
 
-	t.Logf("Creating Elasticsearch CR: %v", cr)
-
-	cleanupOpts := &test.CleanupOptions{
-		TestContext:   ctx,
-		Timeout:       cleanupTimeout,
-		RetryInterval: cleanupRetryInterval,
-	}
-
-	err = f.Client.Create(context.TODO(), cr, cleanupOpts)
```

Comment:
Why does this hunk eliminate the cleanup options? Does this imply no cleanup of CRs on test tear-down?

Reply:
I think the client from controller-time does not support the cleanup Opts. Instead, the cleanup resources are in the `AfterEach` function.